opensrp-chw/src/drc/assets/json.form-fr/family_kit.json

```diff
@@ -0,0 +1,122 @@
+{
```

Comment:
Add this family kit to the main asset folder if no other family_kit.json file exists